compiler/modules/print.llvm/llvm.bal

```diff
@@ -1442,27 +1440,31 @@ function typeToString(RetType ty, Context context, boolean forceInline=false) re
         }
     }
     else if ty is StructType {
-        var data = context.getStructName(ty);
+        string? tyName = ty.name;
+        Type[] elementTypes = ty.elementTypes;
+        if tyName != () {
+            elementTypes = context.getNamedStructBody(ty);
+        }
         if !forceInline {
```

Comment:
This ought to go before the assignment to elementTypes (since it does not depend on that)